### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by azat-co**
CC @AnnJohn @vicfriedman

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+
+By now you should now how to access CLI arguments and use `setTimeout()`. Let's get to building the Timer app!
+
+![](timer.png)
+
+## Instructions
+
+1. Create a CLI app which takes number of seconds in the following format: `ns` for n seconds, e.g., and `5s` for five seconds. The command for 5 seconds must look like `$ node timer 5s`.
+2. Implement the timer by printing the number of seconds left on a new line. For example, if you passed `10s` as your argument, you should see the text `"Left: 10s"`, `"Left: 9s"`, etc. printed to the console.
+3. Enhance the app by adding minutes in the following format nmin, e.g., 1min for 1 minute. The command for 1 minute must look like `$ node timer 1min`.
```
Comment (victhevenot) [inline README.md]: remove `$`

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+
+1. Implement Node timer app which works from CLI (take time and counts down like a kitchen timer)
+2. Implement taking the CLI arguments
+3. Implement the async sleep
+4. Implement the current coutdown 
+
+
+## Introduction
+
+By now you should now how to access CLI arguments and use `setTimeout()`. Let's get to building the Timer app!
```
Comment (victhevenot) [inline README.md:16]: good, but give more detail about the timer app - what does it do?

**Comment by victhevenot** [comment]
we typically like to provide students with all the the files they'll need to complete a lab. it would be great to start including the files for them to start coding their solutions.

**Comment by victhevenot** [comment]
this lab should be a culmination of all the things theyve learned in this unit - but you're asking them to use  `setInterval` and `clearInterval` which they haven't used yet. They also don't get to use the `global` object here, or the `process` object they spent so much time learning. Is there a way to incorporate those skills?

**Comment by azat-co** [comment]
updated CC @AnnJohn @vicfriedman

**Pull request merged**